code/processes/wdb.q

```diff
@@ -205,11 +205,17 @@ endofdaysave:{[dir;pt]
 
 /- add entries to dictionary of callbacks. if timeout has expired or d now contains all expected rows then it releases each waiting process
 handler:{
-	.wdb.d[.z.w]:x;
+	/-insert process reload outcome into .wdb.d
+	if[not .z.w in  key .wdb.d;
+        	.wdb.d[.z.w]:x];
+	if[(not .wdb.d[.z.w]`status);
+		.lg.o[`reload;string[.wdb.d[.z.w]`process]," reload ", string[.wdb.d[.z.w]`result]]];
 	if[(.proc.cp[]>.wdb.timeouttime) or (count[.wdb.d]=.wdb.countreload);
 		.lg.o[`handler;"releasing processes"];
-		.wdb.flushend[];
-		.wdb.d:()!()];
+		.lg.o[`reload;string[count select from .wdb.d where status=1]," out of ", string[count .wdb.d]," processes successfully reloaded"];
+		.wdb.flushend[]];
+	/-delete .wdb.d when reloads completed
+	if[.wdb.reloadcomplete;delete d from `.wdb];
```

Comment:
This will delete the whole thing which we don't want - we just want to delete the contents of .wdb.d
delete from `.wdb.d

Reply:
Apologies misread your last comment have changed this to a delete from

Reply:
Both of those look fine